packages/apidom-ls/src/services/validation/linter-functions.ts

```diff
@@ -1157,4 +1157,78 @@ export const standardLinterfunctions: FunctionItem[] = [
       return true;
     },
   },
+  {
+    functionName: 'apilintSecurityScopeResolved',
+    function: (element: Element): boolean => {
+      const api = root(element);
+      const securityDefinitions = isObject(api) && api.get('securityDefinitions');
+      if (!securityDefinitions || !isObject(securityDefinitions)) return true;
+
+      const securityRequirements = element.toValue() as unknown;
```

Comment:
Perhaps we could do this to make sure it's an Object Element?
```suggestion
      const scopeExists: boolean[] = [];
      if (isObject(element)) {
         const securityRequirements = element.toValue()
      }
```